src/data/syncset/node.rs

```diff
@@ -0,0 +1,556 @@
+use crate::crypto::hash::{hash, Digest};
+
+use super::errors::*;
+use super::path::*;
+use super::Syncable;
+
+use std::cell::{Cell, RefCell};
```

Comment:
Please use the same ordering as the rest of the codebase:
standard imports first, then crate imports and then an import group per external crate used.
All groups should also be sorted alphabetically.